src/comments/interfaces/comments.repository.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { Comment, User } from '@prisma/client';
+
+export interface ICommentsRepository {
+  findCommentById(id: number): Promise<Comment | null>;
+
+  createComment(data: {
+    content: string;
+    userId: number;
+    postId: number;
+    parentId: number | null;
+  }): Promise<Comment>;
+  findComments(
+    postId: number,
+    parentId?: number | null,
+  ): Promise<(Comment & { user: User })[]>;
+  updateComment(id: number, content: string): Promise<Comment>;
+  deleteComment(id: number): Promise<void>;
```

Comment:
comments.service.interface에는 메서드 선언마다 개행이 있어서 동일하게 맞추는게 어떤가 합니다

```suggestion
  createComment(data: {
    content: string;
    userId: number;
    postId: number;
    parentId: number | null;
  }): Promise<Comment>;
  
  findComments(
    postId: number,
    parentId?: number | null,
  ): Promise<(Comment & { user: User })[]>;
  
  updateComment(id: number, content: string): Promise<Comment>;
  
  deleteComment(id: number): Promise<void>;
```